Procfile

```diff
@@ -0,0 +1,5 @@
+# Foreman Procfile. Start all dev server processes with: `foreman start`
```

Comment:
🎉 !

Reply:
Starting and stopping `webpack-dev-server` manually in a separate terminal every time you want to run `rails s`... nobody's got time for that, right?

![](https://media.giphy.com/media/ZEge4UhzUEbzW/giphy.gif)

Reply:
I did the same on a previous draft PR introducing webpacker!